.github/workflows/test-and-deploy.yml

```diff
@@ -17,6 +17,11 @@ jobs:
         with:
           persist-credentials: false
 
+      - name: Install Node.js 💻
+        uses: actions/setup-node@v2
+        with:
+          node-version: 10
```

Comment:
Node 10 is [EOL](https://nodejs.org/en/about/releases/). Any reason not to start with at least 12 or, better yet, 14?